src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue17123.cs

```diff
@@ -0,0 +1,43 @@
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+    public class Issue17123 : _IssuesUITest 
+	{
+        public override string Issue => "TableSection and TableRoot Title properties are displayed as lower case";
+
+        public Issue17123(TestDevice device) : base(device)
+        {
+        }
+
+        [Test]
+		[Category(UITestCategories.TableView)]
+		public void ValidateTableViewTitles()
+        {
+			App.WaitForElement("TableView");
+			var tableViewRootText = App.FindElement("TableRootLabel").GetText();
+			if (string.IsNullOrEmpty(tableViewRootText))
+			{
+				Assert.Fail("Table root text is does not match expected");
+			}
+			else
+			{
+				Assert.That(tableViewRootText, Is.EqualTo("TableRoot Title Text"));
+			}
+
+			var tableViewSelectionText = App.FindElement("TableSectionLabel").GetText();
+			if (string.IsNullOrEmpty(tableViewSelectionText))
+			{
+				Assert.Fail("Table selection text is does not match expected");
+			}
+			else
+			{
+				Assert.That(tableViewSelectionText, Is.EqualTo("TableSection Title Text"));
+			}
+
+			VerifyScreenshot();
```

Comment:
There are some pending snapshots, already available in the latest build:
![image](https://github.com/user-attachments/assets/ddf91903-17df-4856-9830-ce6c83f3e826)

Could you commit the images?

Reply:
@jsuarezruiz I have added the image for iOS. Could you please review it?